src/core/core.scale.js

```diff
@@ -704,7 +715,7 @@ module.exports = function(Chart) {
 					var isLeft = options.position === 'left';
 					scaleLabelX = isLeft ? me.left + (scaleLabelFont.size / 2) : me.right - (scaleLabelFont.size / 2);
 					scaleLabelY = me.top + ((me.bottom - me.top) / 2);
-					rotation = isLeft ? -0.5 * Math.PI : 0.5 * Math.PI;
+					rotation = -0.5 * Math.PI;
```

Comment:
I think that the old version is better, and the labels should have their bottom along the side of the chart.